ReactWindows/ReactNative.Shared.Tests/Bridge/ReactBridgeTests.cs

```diff
@@ -197,6 +198,301 @@ public void ReactBridge_ValidJavaScriptResponse()
             }
         }
 
+        [Test]
+        public async Task SingleReturnCallNoArgs()
```

Comment:
>SingleReturnCallNoArgs [](start = 26, length = 22)

Looks like there is a naming convention for tests that prefixes the class name being tested.

Reply:
that's a deficiency of the Test Explorer UI that's being resolved in an upcoming VS2017 update (or maybe it's already there and not on by default?). Every other test runner (include dotnet CLI) includes the full type of the enclosing test fixture.

Reply:
Still nice to remain consistent for now...

---
In reply to: [185671193](https://github.com/Microsoft/react-native-windows/pull/1726#discussion_r185671193) [](ancestors = 185671193)